pkg/controller/daemon/controller.go

```diff
@@ -572,10 +572,34 @@ func nodeShouldRunDaemonPod(node *api.Node, ds *extensions.DaemonSet) bool {
 	// TODO(mikedanese): remove this once daemonpods forgive nodes
 	shouldRun = shouldRun && api.IsNodeReady(node)
 
-	// If the node is unschedulable, don't run it
-	// TODO(mikedanese): remove this once we have the right node admitance levels.
-	// See https://github.com/kubernetes/kubernetes/issues/17297#issuecomment-156857375.
-	shouldRun = shouldRun && !node.Spec.Unschedulable
+	usedResource := api.ResourceList{}
```

Comment:
Please add a comment pointing to the part of the Kubelet code that this code should be kept in sync with, and also mention that it will eventually be factored into a common library when #12744 is finished.